docs/operator-guides/refreshes.md

```diff
@@ -0,0 +1,128 @@
+- The source you are syncing from is unreliable and you wish to re-attempt the sync
+- Something has gone wrong with your sync, and you wish to try again
+
+Refresh syncs do not create data downtime. While the refresh is running, the previous data in your destination remains available to query. Only at the successful conclusion of a refresh sync with the newly refreshed data be swapped or merged into place. This is an improvement from the previous "Reset Syncs" that Airbyte had in earlier iterations.
+
+When performing a Refresh Sync, any records you’ve already imported will be imported again for the stream(s) being refreshed (assuming they are still present within the source), leading to a slower and more expensive sync than you might be used to. Refresh syncs will block any scheduled syncs from occurring until they are complete. During a Refresh Sync, the streams that are not being refreshed will also sync normally in order to lower data latency . All refresh syncs are resumable, if the underlying source and destination support [checkpointing](/understanding-airbyte/airbyte-protocol/#state--checkpointing).
+
+Depending on the sync mode you’ve selected, you’ll be able to choose between the following refresh options:
+
+| Stream Sync Mode               | Refresh stream and remove records | Refresh stream and retain records | Clear     |
+| ------------------------------ | --------------------------------- | --------------------------------- | --------- |
+| Incremental \| Append + Dedupe | Available                         | Available                         | Available |
+| Full Refresh \| Overwrite      | Not available                     | Not available                     | Available |
+| Full Refresh \| Append         | Not available                     | Not available                     | Available |
+| Incremental \| Append          | Not available                     | Available                         | Available |
+
+For supported destinations, and depending on the sync mode you have selected, there are two types of refreshes - "retaining history" and "removing history". Both kinds of Refresh Syncs can be triggered from the UI and via the Airbyte API at either the stream or connection levels (although both might not be available depending on your sync mode).
+
+![Are you sure you want to refresh data](../.gitbook/assets/refreshes/are-you-sure-refresh.png)
+
+## Refresh and Remove Records
+
+Refresh and Remove Records Syncs will only display data in the destination’s final tables that was newly synced since the refresh was initiated. This is the simplest type of refresh, and leads to table mirroring, if you are using an overwrite sync mode.
+
+:::warning
+
+Not all sources keep their history forever. If you perform a Refresh and Remove Records Sync, and your source does not retain all of its records, this may lead to data loss.
```

Comment:
```suggestion
Not all sources keep their history forever. If you perform a Refresh and Remove Records Sync, and your source does not retain all of its records, this may lead to data loss. If you need to keep all history, then we recommend using `Refresh and Retain Records`. If that is not an option, then you should back up the data in your destination before running the Refresh and Remove Records.
```